src/main/java/uk/gov/pay/adminusers/model/Service.java

```diff
@@ -38,6 +38,7 @@
     private boolean internal;
     private boolean archived;
     private boolean agentInitiatedMotoEnabled;
+    private PSPTestAccountStage pspTestAccountStage;
```

Comment:
Can we name this to `currentPspTestAccountStage` to be consistent with DB column name & ServiceEntity field name ?